pkg/cloud/awsprovider.go

```diff
@@ -1720,9 +1735,22 @@ func (aws *AWS) GetOrphanedResources() ([]OrphanedResource, error) {
 		if aws.isAddressOrphaned(address) {
 			cost := AWSHourlyPublicIPCost * timeutil.HoursPerMonth
 
+			desc := map[string]string{}
+			for _, tag := range address.Tags {
+				if tag.Key == nil {
+					continue
+				}
+				if tag.Value == nil {
+					continue
+				}
```

Comment:
Is there a valid case where `tag.Key != nil` but `tag.Value == nil`?

Reply:
I'm not sure. The tags are coming directly from AWS's API so its entirely possible. Sounds like it could be worth doing `desc[*tag.Key] = ""` if value is empty

Reply:
I think we should roll with that idea!